app/policies/presentation_policy.rb

```diff
@@ -6,4 +6,12 @@ def create?
   def new?
     create?
   end
+
+  def update?
+    person.present? && (person.admin? || person.presentations.include?(record))
```

Comment:
This is purely because of the ? convention. The method must be named `.update?` and having a non boolean return value defies those conventions.